engine/.gitignore

```diff
@@ -0,0 +1 @@
+megalinter-reports/
```

Comment:
Same as above, let's have a root .gitignore that has this and not one per folder